subsys/bluetooth/controller/Kconfig

```diff
@@ -803,7 +803,6 @@ config BT_CTLR_TX_RETRY_DISABLE
 
 config BT_CTLR_CONN_RSSI
 	bool "Connection RSSI"
-	default y if BT_HCI_RAW
```

Comment:
Support for this feature is mandatory if `BT_CONN` is enabled. Just removing the `default y` will lead to more builds with a configuration that cannot pass qualification (cf. #22873).

If we touch this, I would prefer removing the `default y` here and adding `select BT_CTLR_CONN_RSSI if BT_CONN && BT_HCI_RAW` to `BT_CTLR` to close the issue mentioned above.

Reply:
@wopu-ot Thanks. I didn't catch that this was also used for the Read RSSI command. I have split this kconfig option now and put the `default y if BT_HCI_RAW` back.
The RSSI events shouldn't be enabled by default, especially since it is not completely implemented and is not generated an HCI (vendor specific) event.